bugzilla.js

```diff
@@ -59,9 +74,27 @@ function addChoice(form, value, text) {
       [bugId]: categorization,
     });
   };
+  qa_submit.onclick = function(e) {
+    let qa;
+    if(qaChoice.checked){
+      qa = "qa";
+    } else if (noqaChoice.checked){
+      qa = "noqa";
+    } else {
+      alert('You need to select something!');
+    }
+
+    browser.storage.sync.set({
```

Comment:
This is storing the labels as the labels for regression/no-regression/no-bug, we should store them separately or it's going to be hard to download one or the other.